VirtoCommerce.Storefront.IntegrationTests/Cart/ReadingTests.cs

```diff
@@ -29,6 +71,29 @@ public void GetCartItemsCount_IfCartHasItems_ShouldReturnExactItemsCount()
         {
         }
 
+        private ShoppingCart GetNewCart()
```

Comment:
Need to move this to "object builders" or register class with fixture in ctor of base class.

Reply:
removed, unnecessary